specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/privateEndpoint.json

```diff
@@ -50,11 +50,11 @@
             "description": "The name of the resource group."
           },
           {
-            "name": "interfaceEndpointName",
+            "name": "privateEndpointName",
             "in": "path",
             "required": true,
             "type": "string",
-            "description": "The name of the interface endpoint."
+            "description": "The name of the private endpoint."

```

Comment:
Please add the "default" response to model errors, i.e. :
``
          "default": {
            "description": "Error",
            "schema": {
              "$ref": "./network.json#/definitions/Error"
            }
          }
``